.vscode/settings.json

```diff
@@ -5,22 +5,27 @@
 		"*.json": true,
 		"CODEOWNERS": true,
 		"dist": true,
+		"README.md": true,
 		"LICENSE": true,
 		"tsconfig.*": true,
 		"node_modules": true,
-		"*.code-workspace": true
+		"*.code-workspace": true,
+		"*.config.ts": true,
+		"coverage": true
 	},
 	"search.exclude": {
```

Comment:
These were set to false so they could be opened via VSCode `CTRL + SHIFT + P` rather than using a terminal window. I do not use this but some do. I think these should remain set to false